src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -339,6 +339,23 @@ public void RemoveComment(Comment comment)
             AddDomainEvent(new CommentRemovedEvent(Plant, ObjectGuid));
         }
 
+        public void Cancel(Person caller)
```

Comment:
Rename to CancelIPO to keep consistency in naming? Or change SignIPO to Sign, etc, for all relevant places